### DISCUSSION TITLE
How to display ghcr packages in repository

### DISCUSSION THREAD

**Discussion by LesnyRumcajs** [community/community#19254] [Packages] [answered]
I would like to display packages in the repository, e.g. like [here](https://docs.github.com/en/github-ae@latest/packages/learn-github-packages/viewing-packages)
![image](https://user-images.githubusercontent.com/9642092/175237516-ab5c1b0b-00ab-4ef5-a69b-0a9119b97073.png)

Where can I enable this tab? My packages are on `ghcr.io`. They are already public.

**Comment by nickserv** [comment, marked as answer]
Repository > About > Edit repository settings (gear icon) > Include in the home page > Packages > Save changes

**Reply by LesnyRumcajs** [reply]
Nice, thank you!
